amenity-points.mss

```diff
@@ -2901,6 +2901,39 @@
     marker-file: url('symbols/transport_slipway.p.20.svg');
     marker-fill: @transportation-icon;
   }
+
+  [feature = 'attraction_water_slide'] {
+    [zoom >= 16] {
+      [zoom >= 17] {
```

Comment:
z17+ condition is needless, because z16+ covers wider case. I would go with just z17+, as these elements are not that big.